azkaban-common/src/test/java/azkaban/server/session/SessionCacheTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.server.session;
+
+import azkaban.user.User;
+import azkaban.utils.Props;
+import azkaban.utils.PropsUtils;
+import org.junit.Assert;
```

Comment:
Please use AssertJ for new unit tests.
see https://github.com/azkaban/azkaban/blob/master/CONTRIBUTING.md
>  Use AssertJ as the assertion library in unit tests.

Reply:
updated